sdk/ruby/lib/chain/account.rb

```diff
@@ -79,15 +83,17 @@ def create_receiver(opts)
       # Creates new receivers under the specified accounts.
       #
       # @param opts_list [Array<Hash>] Array of options hashes. See {#create_receiver} for a description of the hash structure.
-      # @return [BatchResponse]
+      # @return [BatchResponse<Receiver>]
       def create_receiver_batch(opts_list)
         client.conn.batch_request('create-account-receiver', opts_list) { |item| Receiver.new(item) }
       end
 
-      # @param [Hash] query
+      # @param [Hash] opts Filtering information
+      # @option opts [String] filter Filter string, see {https://chain.com/docs/core/build-applications/queries}
+      # @option opts [Array<String>] filter_params Parameter values for filter string (if needed)
```

Comment:
You can do arrays of integers or strings. Dunno if you can do `Array<String|Integer>`, but `Array<Object>` is okay too.